src/conversion_helper.js

```diff
@@ -758,34 +780,66 @@ function filterConversions(potentialConversions) {
 /*
   Input: imperial conversions
     [
-      { "imperial" : { "number" : 10000, "unit" : " miles" } },
-      { "imperial" : { "number" : 30, "unit" : " mpg" } }
+      { "imperial" : { "numbers" : 10000, "unit" : " miles" } },
+      { "imperial" : { "numbers" : 30, "unit" : " mpg" } }
     ]
   Output: metric and imperial conversions
     [
       { 
         "imperial" : 
-          { "number" : 10000, "unit" : " miles" }, 
+          { "numbers" : 10000, "unit" : " miles" }, 
         "metric": 
-          { "number" : 16093.44, "unit" : " km" } 
+          { "numbers" : 16093.44, "unit" : " km" } 
       },
       { 
         "imperial" : 
-          { "number" : 30, "unit" : " mpg" }, 
+          { "numbers" : 30, "unit" : " mpg" }, 
         "metric" : [
-          { "number" : 12.7543, "unit" : " km/L" },
-          { "number" : 7.84049, "unit" : " L/100km" } 
+          { "numbers" : 12.7543, "unit" : " km/L" },
+          { "numbers" : 7.84049, "unit" : " L/100km" } 
```

Comment:
Just to confirm my understanding: the "numbers" key will always contain an array (even if it's an array of 1) is that correct?

Reply:
Yes.